tests/fixtures/analysis_form/analysis_form.json

```diff
@@ -66,39 +62,23 @@
       }
     },
     {
-      "type": "datetime",
-      "name": "record_a_note/acme_timestamp",
-      "path": [
-        "record_a_note",
-        "acme_timestamp"
-      ],
-      "label": [
-        "Transcription Timestamp"
-      ],
-      "source": "record_a_note"
-    },
-    {
-      "type": "text",
+      "type": "qual_text",
       "name": "name_of_clerk/comment",
       "path": [
         "clerk_details-name_of_clerk",
         "comment"
       ],
-      "label": [
-        "Comment on the name of the clerk"
-      ],
+      "label": "Comment on the name of the clerk",
```

Comment:
To match KPI, which does not yet support multi-language labels for qualitative analysis questions